### DISCUSSION TITLE
I need help with the sendcoins command

### DISCUSSION THREAD

**Discussion by arthur-feral** [lightningnetwork/lnd#8219] [Troubleshooting]
Hello,

I have a lnd node with a default wallet configured and unlocked.

I want to send some sats on-chain and for that I am using the sendcoins command.

The problem is that the output amount calculated seems to include a crazy amount of fees:

[/lnrpc.Lightning/SendCoins]: insufficient input to create sweep tx: input_sum=0.00341786 BTC, output_sum=7.47600000 BTC

I am struggling to understand why.

Do you have any idea if this comes from a misconfiguration of the node?

Thanks for your help!

**Comment by hieblmi** [comment]
Hi @arthur-feral, thanks for starting this discussion.

If you try to send a certain amount to an address and would like to control the fee rate your command could look something like this:
`lncli sendcoins --addr <dest_addr> --amt <amt_to_send> --sat_per_vbyte <sat/vbyte>`

This shouldn't come from a misconfiguration of your node. Are you trying to send more funds than you have available in your wallet?
What is the command you are using?

**Comment by arthur-feral** [comment]
Hello, Thanks for your answer.

so the output of 

```bash
lncli listchannels
```
```
{
    "channels":  []
}
```

```bash
lncli pendingchannels
```
is 
```
lncli pendingchannels
{
    "total_limbo_balance":  "0",
    "pending_open_channels":  [],
    "pending_closing_channels":  [],
    "pending_force_closing_channels":  [],
    "waiting_close_channels":  []
}
```

```bash
lncli walletbalance
```
```
{
    "total_balance":  "341786",
    "confirmed_balance":  "341786",
    "unconfirmed_balance":  "0",
    "locked_balance":  "0",
    "reserved_balance_anchor_chan":  "0",
    "account_balance":  {
        "default":  {
            "confirmed_balance":  "341786",
            "unconfirmed_balance":  "0"
        }
    }
}
```

now if I do this:
```bash
lncli sendcoins --addr xxx --amt 10000 --sat_per_vbyte 50
```

I get
```
[lncli] rpc error: code = Unknown desc = manual fee rate input of 12500 sat/kw is too low to be accepted into the mempool or relayed to the network
```

What would you suggest?

**Reply by hieblmi** [reply]
Is your intent to sweep all confirmed funds to an address? Could you try it with `sat_per_vbyte` instead of `conf_target`?

**Comment by arthur-feral** [comment]
The command returning the error with the crazy fees is

```bash
lncli sendcoins --sweepall --conf_target 800 bc1xxx
```

**Reply by hieblmi** [reply]
The result of this seems really off, probably due to your local mempool size and the current fee environment.

**Comment by arthur-feral** [comment]
@hieblmi Thanks for your answers and help.
Eventually, I've asked chatgpt what was wrong, gave some context and quoted your answer, and it gave me a fix for the bitcoind config. All good, it works normally now.

**Reply by hieblmi** [reply]
Nice that you could resolve it. May I ask what the suggested fix is that works?

**Reply by arthur-feral** [reply]
the value for `minrelaytxfee` was incorrect and changing it to something like `0.00001` fixed my issue

**Comment by Zara2911** [comment]
buy btc
